app/views/AdminView/components/Appendices/columnDefs.ts

```diff
@@ -3,12 +3,12 @@ import { ColDef } from '@ag-grid-community/core';
 
 const columnDefs: ColDef[] = [
   {
-    headerName: 'Name',
+    headerName: 'Tamplate Name',
     field: 'template.name',
   },
   {
-    headerName: 'Org',
-    field: 'template.organization',
+    headerName: 'Project',
+    cellRenderer: ({ data }) => data.project.name || 'default appendix text',
```

Comment:
Not sure this looks right. Maybe 'default appendix text' -> '(default appendix text)' and if that doesn't look right we'll figure something else out later